docs/api/structures/window-open-handler-response.md

```diff
@@ -0,0 +1,20 @@
+# WindowOpenHandlerResponse Object
+
+* `action` string - Can be `allow` or `deny`. Controls whether new window should be created.
+* `overrideBrowserWindowOptions` BrowserWindowConstructorOptions (optional) - Allows customization of the created window.
+* `outlivesOpener` boolean (optional) - Controls whether created window should stay open when its opener was closed.
+* `overrideWindowCreationFunction` (options: BrowserWindowConstructorOptions, wc?: WebContents) => BrowserWindow (optional) - Allows to manually control creation of window.
```

Comment:
```suggestion
* `createWindow` (options: BrowserWindowConstructorOptions) => BrowserWindow (optional) - If specified, will be called instead of `new BrowserWindow` to create the new child window. This can be used for example to have the new window open as a BrowserView instead of in a separate window.
```

Let's rename this and simplify it a little—we can pass in a pure `options` structure, and include the `webContents` parameter in it.